### DISCUSSION TITLE
Forking Cypress Postinstall Error

### DISCUSSION THREAD

**Discussion by tgf-coder** [cypress-io/cypress#14869] [Questions and Help]
I have a gitlab repository with a fork of cypress. I am trying to do a git+ssh to install cypress on my node project but am getting this error:

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! cypress@6.3.0 postinstall: `./scripts/run-if-not-ci.sh yarn build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the cypress@6.3.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

How can I fix this?

**Comment by tgf-coder** [comment]
I fixed it by making the postinstall use \ instead of / because I am working with a windows computer.

**Comment by jennifer-shehane** [comment]
See our Contributing guide: https://github.com/cypress-io/cypress/blob/develop/CONTRIBUTING.md#getting-started There are some notes for Windows users in there.
